templates/embed/Article.coffee

```diff
@@ -28,7 +28,7 @@ class Article
         Carnival.removeClass(@element, 'commenting')
 
   fetchComments: ->
-    Carnival.get('/comments?article=' + @id, (data) =>
+    Carnival.get('@{CommentsR}?article=' + @id, (data) =>
```

Comment:
Yes, because of [this](https://github.com/thoughtbot/carnival/blob/master/Foundation.hs#L55), it will use [this](https://github.com/thoughtbot/carnival/blob/master/config/settings.yml#L4), [this](https://github.com/thoughtbot/carnival/blob/master/config/settings.yml#L15), and [this](https://github.com/thoughtbot/carnival/blob/master/config/settings.yml#L19) depending on environment.